modules/support_ticket/config/optional/field.field.support_ticket.ticket.field_assigned_to.yml

```diff
@@ -0,0 +1,30 @@
+uuid: 9d53bedd-ebbc-4d6a-ba32-50d2ddd45c83
```

Comment:
I wondered about that -- these are from when I export configuration. Should I completely strip the uuid line in all config files, or does some sort of filler go there? (I think I saw something like "uuid: UUID" or something like that in some modules).